hoomd/hpmc/update.py

```diff
@@ -845,11 +848,25 @@ class QuickCompress(Updater):
     box move sizes will be uniformly distributed between ``min_scale`` and 1.0
     (with no consideration of ``min_move_size``).
 
+    When using a `BoxVariant` for `target_box`, `complete` returns `True` if the
+    current simulation box is equal to the box corresponding to `target_box`
+    evaluated at the current timestep and there are no overlaps in the system.
+    To ensure the updater has compressed the system to the final target box, use
+    a condition that checks both the `complete` attribute of the updater and the
+    simulation timestep. For example::
+
+        target_box = hoomd.variant.box.InverseVolumeRamp(
+            sim.state.box, sim.state.box.volume / 2, 0, 10000)
```

Comment:
```suggestion
        target_box = hoomd.variant.box.InverseVolumeRamp(
            sim.state.box, sim.state.box.volume / 2, 0, 1_000)
```
We should put this in Sybil, but I see that there are no other Sybil examples in this file. I'll make a separate PR for a large batch of Sybil updates.

For now, let's shorten the steps so the doc example will not run for long when it is added to Sybil.